CrowsZA/functions/fn_spawnIEDClutter.sqf

```diff
@@ -114,7 +114,7 @@ for "_i" from 1 to _clutterAmount do {
 	};
 	_clutter = _clutter createVehicle _safePos;
 	_clutter setDir (random 360);
-
+	[_clutter, false] remoteExec ["enableSimulationGlobal", 2];
```

Comment:
Have you tested if it works when you just run it locally, instead of telling the server to run it? 
```sqf
_myObject enableSimulationGlobal [false](https://community.bistudio.com/wiki/false);
```
From wiki: 
![image](https://user-images.githubusercontent.com/7889925/198040980-d7b9d299-ae6c-4edb-9bcc-31ba0e9b20a4.png)
 
 Thinking if we don't need to call it from server, it would be better to just call it locally and let it propagate itself across.

Reply:
Ah, no - I just copied the code from drawBuild. Will give this a try!

Reply:
Seems to work fine - haven't been able to test it across multiple clients though